lib/Dancer2/Plugin/UnicodeNormalize.pm

```diff
@@ -6,18 +6,17 @@ package Dancer2::Plugin::UnicodeNormalize;
     $Dancer2::Plugin::UnicodeNormalize::VERSION = '0.033';
 }
 
-use Dancer2 0.14300;
 use Dancer2::Plugin;
 use Unicode::Normalize;
 
 on_plugin_import {
     my $dsl = shift;
-    my $settings = plugin_setting();
 
     $dsl->app->add_hook(
         Dancer2::Core::Hook->new(
             name => 'before',
             code => sub {
+                my $settings = plugin_setting;
```

Comment:
Scratch that, just tested against https://github.com/PerlDancer/Dancer2/pull/1010 and all seems well.

Since this breaks on existing installations (as far as I can tell) I guess we'll need to coordinate the release of this plugin with Dancer2? Or do we need the plugins resolved beforehand? I'm thinking of requiring the upcoming version of Dancer2::Plugin for this release.

I think we also need to change instances of `$dsl->request` to `$dsl->app->request` too?